models/engine/file_storage.py

```diff
@@ -66,5 +66,18 @@ def delete(self, obj=None):
                 del self.__objects[key]
 
     def close(self):
-        """call reload() method for deserializing the JSON file to objects"""
+        """Call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """Method to retrieve one ojbect"""
```

Comment:
Maybe add a little bit more information to the doc string about what the function returns and why.